condensedinlinepanel/edit_handlers.py

```diff
@@ -156,7 +156,7 @@ def required_formsets(cls):
 
 
 class CondensedInlinePanel(object):
-    def __init__(self, relation_name, panels=None, heading='', label='', help_text='', min_num=None, max_num=None, card_header_from_field=None, card_header_from_js=None, card_header_from_js_safe=None):
+    def __init__(self, relation_name, panels=None, heading='', label='', help_text='', min_num=None, max_num=None, card_header_from_field=None, card_header_from_js=None, card_header_from_js_safe=None, new_card_header_text=""):
         self.relation_name = relation_name
         self.panels = panels
         self.heading = heading or label
```

Comment:
I'm not sure using `label` for `heading` here under these conditions is very intuitive now that we define a more sensible default for heading lower down. I think we're just doing this for the sake of backwards compatibility, but I think it would be better instead to decouple the two completely and include some guidance in the release notes instead. Any thoughts?

Reply:
While I think this panel type isn't being used widely, I tend to lean towards backwards compatibility for any changes unless entirely unavoidable.

Plus, a lot of what I've ended up doing here, even with the defaults we've now added, I actually want to move to the standard wagtail inline panel, so keeping the changes here, and there in sync so the two are pretty much drop in replacements for each other (apart from card_header args) is useful. I've already started with: https://github.com/wagtail/wagtail/pull/3765

Reply:
I'm happy with that. Might be a good idea to mark it with a TODO comment or something though, to make that clear. Maybe somethings like:

```
# TODO: label is used below for backwards compatibility. We may want to rethink this later.
self.heading = heading or label
```

Reply:
done :)